packages/ckeditor5-widget/src/utils.ts

```diff
@@ -275,18 +276,24 @@ export function getLabel( element: ViewElement ): string {
  *
  * @param options Additional options.
  * @param options.label Editable's label used by assistive technologies (e.g. screen readers).
+ * @param options.withAriaRole Whether to add the role="textbox" attribute on the editable. Defaults to `true`.
  * @returns Returns the same element that was provided in the `editable` parameter
  */
 export function toWidgetEditable(
 	editable: ViewEditableElement,
 	writer: DowncastWriter,
 	options: {
 		label?: string;
+		withAriaRole?: boolean;
```

Comment:
I'd suggest:
```suggestion
		ariaRole?: string = 'textbox';
```

Reply:
Or maybe as other aria role does not make sense we could name it simply `ariaRole` without "with"?